src/modules/mc_pos_control_multiplatform/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ px4_add_module(
 	MODULE modules__mc_pos_control_multiplatform
 	MAIN mc_pos_control_m
 	COMPILE_FLAGS
-		-Wframe-larger-than=1200
+		-Wframe-larger-than=1900
```

Comment:
This was for posix_sitl_simple. Should I just change this warning so it only applies to nuttx and qurt?